hip-service/wwwroot/swagger.yaml

```diff
@@ -215,10 +215,10 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Error'
+                $ref: '#/components/schemas/PatientDiscoveryErrResponse'
             application/xml:
               schema:
-                $ref: '#/components/schemas/Error'
+                $ref: '#/components/schemas/PatientDiscoveryErrResponse'
```

Comment:
Why is link err response called `PatientDiscoveryErrResponse`? You can name it something thats common to both link and discovery.